src/ui/common/rich-text-editor/RichTextEditor.js

```diff
@@ -211,6 +211,12 @@ class RichTextEditor extends Component {
       this.quill.clipboard.dangerouslyPasteHTML(html);
     }
     this.txtArea.style.display = this.txtArea.style.display === 'none' ? '' : 'none';
+    const parentTxtarea = this.txtArea.closest('.ql-srccode');
+    if (parentTxtarea.classList.contains('active')) {
```

Comment:
You can just use [`classList.toggle()`](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList#:~:text=div.classList.toggle(%22visible%22)%3B) here.

Reply:
oh yeah I forgot about this. ugh. Thanks @raxkaynan